### DISCUSSION TITLE
Where do Archived tasks go?  How can I import an exported project?

### DISCUSSION THREAD

**Discussion by neko-cha** [stacks-task-manager/stacks2#52] [Q&A] [answered]
I just purchased a full license so am still playing around with the features.  A couple of questions I have are:

1. I archived a task, but it does not appear in the "Archived tasks".  Where did it go?
2. I exported a project because I want to move it to a different workspace, however I can only import from Asana or Trello!

Appreciate if anyone knows the answers to these questions.

Thank you!

**Comment by synapse** [comment, marked as answer]
Hey @neko-cha the archived tasks are all grouped into their own section accessible from the Sidebar.
![Screenshot 2023-11-09 at 12 22 56](https://github.com/stacks-task-manager/stacks2/assets/71137/fe5e8cac-2641-4e45-9004-419eb6d078f4)

N.B.: You can also pin the `Archived task` menu item in case you need to access it more often.

**Comment by neko-cha** [comment]
Thanks for the response.  I just did another test just now and it does seem to be working now which is strange as I previous archived 2 tasks but the "Archived tasks" was empty.  At the time I was also experimenting moving tasks between projects and such so seems maybe I did something weird.
